### DISCUSSION TITLE
Component Library?

### DISCUSSION THREAD

**Discussion by laurentlahmy** [chakra-ui/panda#712] [General]
Hi!

Will this project come with a component library like Chakra UI?
Is the component library based on panda a separate project? 

Thanks for all your contributions.

**Comment by paulm17** [comment]
@laurentlahmy Take a look at https://www.adebayosegun.com/blog/the-future-of-chakra-ui

I think the intention is to develop a next-gen Chakra with all the new projects they have been developing and it includes this one.

**Comment by segunadebayo** [comment]
This project will not come with a component library. It's purely a styling engine you can use to build components.

Chakra UI and its future versions will remain the "component library"
